assignments/objects.js

```diff
@@ -18,27 +18,62 @@ const example = {
 }
 
 // Write your intern objects here:
+const intern1 = {
+  id: 1,
+  name: 'Mitzi',
+  email: 'mmelloy0@psu.edu',
+  gender: 'F',
+}
+
+const intern2 = {
+  id: 2,
+  name: 'Kennan',
+  email: 'kdiben1@tinypic.com',
+  gender: 'M',
+}
 
+const intern3 = {
+  id: 3,
+  name: 'Keven',
+  email: 'knummery2@wikimedia.org',
+  gender: 'M'
+}
+
+const intern4 = {
+  id: 4,
+  name: 'Gannie',
+  email: 'gmartinson3@illinois.edu',
+  gender: 'M'
+}
 
+const intern5 = {
+  id: 5,
+  name: 'Antonietta',
+  email: 'adaine5@samsung.com',
+  gender: 'F',
+}
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(intern1.name)
 // Kennan's ID
-
+console.log(intern2.id)
 // Keven's email
-
+console.log(intern3.email)
 // Gannie's name
-
+console.log(intern4.name)
 // Antonietta's Gender
-
+console.log(intern5.gender)
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
-
```

Comment:
The goal here was to do exactly what you did with the problem below, but you nailed that so no big deal.